protocol-units/da/m1/setup/src/arabica.rs

```diff
@@ -0,0 +1,236 @@
+    }
+
+    pub async fn get_auth_token(&self) -> Result<String, anyhow::Error> {
+        // celestia light auth admin --p2p.network arabica
+        let auth_token = run_command(
+            "celestia",
+            &[
+                "light",
+                "auth",
+                "admin",
+                "--p2p.network",
+                "arabica",
+            ],
+        ).await?.trim().to_string();
+
+        Ok(auth_token)
+    }
+
+    pub async fn setup_celestia(
+        &self,
+        dot_movement: DotMovement,
+        mut config: Config,
+    ) -> Result<Config, anyhow::Error> {
+
+        let mut config = common::celestia::initialize_celestia_config(dot_movement.clone(), config)?;
+		let mut config = common::memseq::initialize_memseq_config(dot_movement.clone(), config)?;
+		let mut config = common::celestia::make_dirs(dot_movement.clone(), config).await?;
+
+        // celestia light init --p2p.network arabica
+        self.celestia_light_init().await?;
+
+        // get the arabica 11 address
+        let address = self.get_arabica_11_address().await?;
+        config.appd.celestia_validator_address.replace(address.clone());
+
+        // get the auth token
+        let auth_token = self.get_auth_token().await?;
+        config.appd.celestia_auth_token.replace(auth_token.clone());
+
+        // create and fund the account
+        self.create_and_fund_account(dot_movement.clone(), config.clone()).await?;
+
+        Ok(config)
+    }
+
+    pub async fn create_and_fund_account(
+        &self,
+		dot_movement: DotMovement,
+		config: Config,
+    ) -> Result<(), anyhow::Error> {
+        
+        /**
+         * #!/bin/bash
+        # Maximum number of retries
+        max_retries=10
+        # Delay in seconds between retries
+        retry_delay=5
+
+        retry_count=0
+        success=false
+
+        while [ $retry_count -lt $max_retries ]; do
+            # Run the curl command
+            response=$(curl -s -X POST 'https://faucet.celestia-arabica-11.com/api/v1/faucet/give_me' \
+                -H 'Content-Type: application/json' \
+                -d "{\"address\": \"$CELESTIA_ADDRESS\", \"chainId\": \"arabica-11\"}")
+            
+            # Process the response with jq
+            txHash=$(echo "$response" | jq -e '.txHash')
+            
+            # Check if jq found the txHash
+            if [ $? -eq 0 ]; then
+                echo "Transaction hash: $txHash"
+                success=true
+                break
+            else
+                echo "Error: txHash field not found in the response." >&2
+                # Increment the retry counter
+                retry_count=$((retry_count+1))
+                # Wait before retrying
+                sleep $retry_delay
+            fi
+        done
+
+        # Check if the operation was successful
+        if [ "$success" = false ]; then
+            echo "Failed to retrieve txHash after $max_retries attempts." >&2
+        fi
+         */
```

Comment:
Any reason for the commented out bash here? I can see you've written this out in rust (yay) but I'd just get rid of the commented out bash. Or if we need it for ref, put it in a README.md.